Documentation/botr/clr-abi.md

```diff
@@ -85,7 +85,9 @@ ARM64-only: When a method returns a structure that is larger than 16 bytes the c
 
 # PInvokes
 
-The convention is that any method with an InlinedCallFrame (either an IL stub or a normal method with an inlined pinvoke) saves/restores all non-volatile integer registers in its prolog/epilog respectively. This is done so that the InlinedCallFrame can just contain a return address, a stack pointer and a frame pointer. Then using just those three it can start a full stack walk using the normal RtlVirtualUnwind.
+The convention is that any method with an InlinedCallFrame (either an IL stub or a normal method with an inlined PInvoke) saves/restores all non-volatile integer registers in its prolog/epilog respectively. This is done so that the InlinedCallFrame can just contain a return address, a stack pointer and a frame pointer. Then using just those three it can start a full stack walk using the normal RtlVirtualUnwind.
+
+When encountering a PInvoke, the JIT will query the VM if the GC transition should be suppressed. If the VM indicates the GC transition is to be suppressed, the PInvoke frame will be omitted in either the IL stub or inlined scenario and a GC Poll will be insert near the unmanaged call site. In an enclosing function contains more than one PInvoke that is to be inlined, but not all have requested a suppression of the GC transition a PInvoke frame will still be constructed for the other inlined PInvokes. 
```

Comment:
"if the GC transition should be suppressed". It might be useful to state how this might occur, such as a mention of the attribute and/or a link to the spec for the same.